apps/marketplace/components/BuyerSpecialist/BuyerSpecialistSelectStage.js

```diff
@@ -109,83 +107,82 @@ export class BuyerSpecialistSelectStage extends Component {
-                messages={{}}
-                onChange={() => this.props.resetSelectedSellers()}
-              />
-            </div>
-            {this.props[this.props.model].openTo === 'all' && (
-              <AUpageAlert as="warning">
-                <AUheading level="2" size="lg">
-                  You are about to invite{' '}
-                  {this.state.totalLabourHireSellers >= 1 ? this.state.totalLabourHireSellers : 'all'} seller
-                  {this.state.totalLabourHireSellers === 1 ? '' : 's'}.
-                </AUheading>
-                <div className={mainStyles.marginTop1}>
-                  <p>
-                    You will need to reply to all seller questions, evaluate all quotes and debrief all unsuccessful
-                    sellers (where requested).
-                  </p>
-                  <p>
-                    You should only invite all sellers if there is a business need for this approach. If you have
-                    questions, please{' '}
-                    <a href="/contact-us" rel="noopener noreferrer" target="_blank">
-                      contact us
-                    </a>
-                    .
-                  </p>
-                </div>
-              </AUpageAlert>
-            )}
-            {this.props[this.props.model].openTo === 'selected' && (
-              <React.Fragment>
-                <SellerSelect
-                  briefId={this.props[this.props.model].id}
-                  label="Seller name"
-                  showSelected={false}
-                  showSearchButton={false}
-                  categories={categories}
-                  onSellerSelect={this.handleSellerSelect}
-                  onSellerCategorySelect={this.handleSellerCategorySelect}
-                  showCategorySelect={false}
-                  notFoundMessage="Seller is not on the Digital Marketplace"
-                  selectedCategory="labourHire"
-                  showSellerCatalogueLink
-                />
-                <br />
-                <SelectedSellersControl
-                  id="selected-sellers"
-                  model={`${this.props.model}.sellers`}
-                  onRemoveClick={sellerCode => this.removeSeller(sellerCode)}
-                />
-              </React.Fragment>
-            )}
+            <RadioList
+              id="openTo"
+              label="Accept responses from:"
+              name="openTo"
+              model={`${this.props.model}.openTo`}
+              options={[
+                {
+                  label: 'Any seller that provides ICT Labour Hire',
+                  value: 'all'
+                },
+                {
+                  label: 'Specific ICT Labour Hire sellers',
+                  value: 'selected'
+                }
+              ]}
+              messages={{}}
+            />
           </div>
-        )}
+          {this.props[this.props.model].openTo === 'all' && (
+            <AUpageAlert as="warning">
```

Comment:
Do you think we should add some space to the bottom of the page alert to separate it from the category select?
```suggestion
            <AUpageAlert as="warning" className={mainStyles.marginBottom3}>
```